src/components/Navbar/styles.scss

```diff
@@ -26,7 +25,7 @@
   .nav-section {
     padding: $tw-spc-base-x-small $tw-spc-base-small;
     .divider {
-      border-top: $tw-sz-dvdr-stroke solid $tw-clr-brdr-tertiary;
+      border-top: 1px solid $tw-clr-brdr-tertiary;
```

Comment:
The `-divider-` zeroheight files seem to have been removed, likely because that particular component is no longer in use. There are no other `1px` tokens to replace this one with.